main.go

```diff
@@ -1,17 +1,14 @@
-package main
+package ironio
```

Comment:
The root package needs to be called `main` as any other name will result in an error once Terraform tries to use the provider.